cpp/src/ray/runtime/abstract_ray_runtime.cc

```diff
@@ -45,13 +44,17 @@ using ray::core::CoreWorkerProcess;
 using ray::core::WorkerType;
 
 std::shared_ptr<AbstractRayRuntime> AbstractRayRuntime::abstract_ray_runtime_ = nullptr;
+std::unique_ptr<ray::gcs::GlobalStateAccessor>
+    AbstractRayRuntime::global_state_accessor_ = nullptr;
 
 std::shared_ptr<AbstractRayRuntime> AbstractRayRuntime::DoInit() {
   std::shared_ptr<AbstractRayRuntime> runtime;
   if (ConfigInternal::Instance().run_mode == RunMode::SINGLE_PROCESS) {
     runtime = std::shared_ptr<AbstractRayRuntime>(new LocalModeRayRuntime());
   } else {
-    ProcessHelper::GetInstance().RayStart(TaskExecutor::ExecuteTask);
+    global_state_accessor_ = nullptr;
+    ProcessHelper::GetInstance().RayStart(TaskExecutor::ExecuteTask,
+                                          &global_state_accessor_);
```

Comment:
passing `gcs_global_state_accessor` to `RayStart` is weird. 
You can get the global state accessor from the CoreWorker instead.

Reply:
The global state accessor is not in CoreWorker, can't get it from coreworker.